src/server/player_sao.cpp

```diff
@@ -367,6 +367,30 @@ void PlayerSAO::setPos(const v3f &pos)
 	m_env->getGameDef()->SendMovePlayer(m_peer_id);
 }
 
+void PlayerSAO::addPos(const v3f &added_pos)
+{
+	// Backward compatibility for older clients
+	if (m_player->protocol_version < 44) {
+		setPos(getBasePosition() + added_pos);
+		return;
+	}
+
+	if(isAttached())
```

Comment:
surely this should be above the other check

Reply:
fixed

Reply:
Since `setPos` also checks `isAttached`, the order doesn't actually matter.